ERROR: Could not build wheels for pycairo

<!--
The issue tracker is a tool to address bugs.
For support questions, please use: https://github.com/kivy/buildozer#support

Before opening a new issue, make sure you do the following:
    * check that your issue isn't already filed: https://github.com/kivy/buildozer/issues
    * prepare a short, runnable example that reproduces the issue
    * make sure to have `log_level = 2` in your `buildozer.spec`
    * reproduce the problem with the latest development version of Kivy
    * double-check that the issue is indeed a bug and not a support request
-->

### Versions

* Python: 3.9
* OS: ubuntu
* Buildozer: 1.5.0

### Description

i was trying to added a package into my kivy application i.e pycairo
so i have added the pycairo into my .spec file

### buildozer.spec
Command:
```
sudo apt install libcairo2-dev pkg-config python3-dev
buildozer android debug
```

Spec file:
```
# (list) Application requirements
# comma separated e.g. requirements = sqlite3,kivy
requirements = python3,kivy,kivymd==1.0.2,requests==2.28.0,urllib3==1.26.14,charset_normalizer==3.0.0,chardet==5.0.0,idna==3.3,PIL,kivy_garden.mapview,msal,msal_extensions,portalocker,olefile,pycairo
```

### Logs

```
Downloading urllib3-1.26.14-py2.py3-none-any.whl (140 kB)
25l     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 0.0/140.6 kB ? eta -:--:--
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 140.6/140.6 kB 42.1 MB/s eta 0:00:00
25hBuilding wheels for collected packages: pycairo, olefile
  Running command Building wheel for pycairo (pyproject.toml)
  Ignoring "sys._home = value" override
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-cpython-310
  creating build/lib.linux-x86_64-cpython-310/cairo
  copying cairo/__init__.py -> build/lib.linux-x86_64-cpython-310/cairo
  copying cairo/__init__.pyi -> build/lib.linux-x86_64-cpython-310/cairo
  copying cairo/py.typed -> build/lib.linux-x86_64-cpython-310/cairo
  running build_ext
  building 'cairo._cairo' extension
  creating build/temp.linux-x86_64-cpython-310
  creating build/temp.linux-x86_64-cpython-310/cairo
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/bufferproxy.c -o build/temp.linux-x86_64-cpython-310/cairo/bufferproxy.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/cairomodule.c -o build/temp.linux-x86_64-cpython-310/cairo/cairomodule.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/context.c -o build/temp.linux-x86_64-cpython-310/cairo/context.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/device.c -o build/temp.linux-x86_64-cpython-310/cairo/device.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/enums.c -o build/temp.linux-x86_64-cpython-310/cairo/enums.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/error.c -o build/temp.linux-x86_64-cpython-310/cairo/error.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/font.c -o build/temp.linux-x86_64-cpython-310/cairo/font.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/glyph.c -o build/temp.linux-x86_64-cpython-310/cairo/glyph.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/matrix.c -o build/temp.linux-x86_64-cpython-310/cairo/matrix.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/misc.c -o build/temp.linux-x86_64-cpython-310/cairo/misc.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/path.c -o build/temp.linux-x86_64-cpython-310/cairo/path.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/pattern.c -o build/temp.linux-x86_64-cpython-310/cairo/pattern.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/rectangle.c -o build/temp.linux-x86_64-cpython-310/cairo/rectangle.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/region.c -o build/temp.linux-x86_64-cpython-310/cairo/region.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  /home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -target aarch64-linux-android21 -fomit-frame-pointer -march=armv8-a -fPIC -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/other_builds/python3/arm64-v8a__ndk_target_21/python3/Include -DANDROID -I/home/vsts/.buildozer/android/platform/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/python-installs/com.zf.sdv.appstore/arm64-v8a/include/python3.1 -fPIC -DPYCAIRO_VERSION_MAJOR=1 -DPYCAIRO_VERSION_MINOR=24 -DPYCAIRO_VERSION_MICRO=0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/home/vsts/work/1/s/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/build/venv/include -I/usr/local/include/python3.10 -c cairo/surface.c -o build/temp.linux-x86_64-cpython-310/cairo/surface.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
  In file included from cairo/surface.c:2280:
  /usr/include/cairo/cairo-xcb.h:46:10: fatal error: 'xcb/xcb.h' file not found

  running build_py
  creating build
  creating build/lib
  creating build/lib/olefile
  copying olefile/__init__.py -> build/lib/olefile
  copying olefile/olefile.py -> build/lib/olefile
  installing to build/bdist.linux-x86_64/wheel
  running install
  running install_lib
  creating build/bdist.linux-x86_64
  creating build/bdist.linux-x86_64/wheel
  creating build/bdist.linux-x86_64/wheel/olefile
  copying build/lib/olefile/__init__.py -> build/bdist.linux-x86_64/wheel/olefile
  copying build/lib/olefile/olefile.py -> build/bdist.linux-x86_64/wheel/olefile
  running install_egg_info
  running egg_info
  writing olefile.egg-info/PKG-INFO
  writing dependency_links to olefile.egg-info/dependency_links.txt
  writing top-level names to olefile.egg-info/top_level.txt
  reading manifest file 'olefile.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  no previously-included directories found matching 'doc/_build'
  adding license file 'LICENSE.txt'
  writing manifest file 'olefile.egg-info/SOURCES.txt'
  Copying olefile.egg-info to build/bdist.linux-x86_64/wheel/olefile-0.46-py3.10.egg-info
  running install_scripts
  creating build/bdist.linux-x86_64/wheel/olefile-0.46.dist-info/WHEEL
  creating '/tmp/pip-wheel-prwdvpyr/.tmp-a3ez_oo9/olefile-0.46-py2.py3-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
  adding 'olefile/__init__.py'
  adding 'olefile/olefile.py'
  adding 'olefile-0.46.dist-info/LICENSE.txt'
  adding 'olefile-0.46.dist-info/METADATA'
  adding 'olefile-0.46.dist-info/WHEEL'
  adding 'olefile-0.46.dist-info/top_level.txt'
  adding 'olefile-0.46.dist-info/RECORD'
  removing build/bdist.linux-x86_64/wheel
  Building wheel for olefile (pyproject.toml) ... 25l25hdone
  Created wheel for olefile: filename=olefile-0.46-py2.py3-none-any.whl size=35417 sha256=4a34dbbc9bf7dfc57c59ce7ac44b3b097a156cf0f343628b1e20efcc8d95259a
  Stored in directory: /home/vsts/.cache/pip/wheels/02/39/c0/9eb1f7a42b4b38f6f333b6314d4ed11c46f12a0f7b78194f0d
Successfully built olefile
Failed to build pycairo
ERROR: Could not build wheels for pycairo, which is required to install pyproject.toml-based projects
```

--- a/pythonforandroid/recipe.py
+++ b/pythonforandroid/recipe.py
@@ -1347,6 +1347,10 @@ class MesonRecipe(PyProjectRecipe):
     meson_version = "1.4.0"
     ninja_version = "1.11.1.1"
 
+    skip_python = False
+    '''If true, skips all Python build and installation steps.
+    Useful for Meson projects written purely in C/C++ without Python bindings.'''
+
     def sanitize_flags(self, *flag_strings):
         return " ".join(flag_strings).strip().split(" ")
 
@@ -1428,7 +1432,8 @@ class MesonRecipe(PyProjectRecipe):
         ]:
             if dep not in self.hostpython_prerequisites:
                 self.hostpython_prerequisites.append(dep)
-        super().build_arch(arch)
+        if not self.skip_python:
+            super().build_arch(arch)
 
 
 class RustCompiledComponentsRecipe(PyProjectRecipe):

--- a/pythonforandroid/recipes/libcairo/__init__.py
+++ b/pythonforandroid/recipes/libcairo/__init__.py
@@ -1,0 +1,85 @@
+from pythonforandroid.recipe import Recipe, MesonRecipe
+from os.path import join, exists
+from pythonforandroid.util import ensure_dir, current_directory
+from pythonforandroid.logger import shprint
+from multiprocessing import cpu_count
+import sh
+
+
+class LibCairoRecipe(MesonRecipe):
+    name = 'libcairo'
+    version = '1.18.4'
+    url = 'https://gitlab.freedesktop.org/cairo/cairo/-/archive/{version}/cairo-{version}.tar.bz2'
+    skip_python = True
+    depends = ["png", "freetype"]
+    patches = ["meson.patch"]
+    built_libraries = {
+        'libcairo.so': 'install/lib',
+        'libpixman-1.so': 'install/lib',
+        'libcairo-script-interpreter.so': 'install/lib'
+    }
+
+    def get_recipe_env(self, arch, **kwargs):
+        env = super().get_recipe_env(arch, **kwargs)
+        cpufeatures = join(self.ctx.ndk.ndk_dir, "sources/android/cpufeatures")
+        lib_dir = join(cpufeatures, "obj", "local", arch.arch)
+        env["CFLAGS"] += f" -I{cpufeatures}"
+        env["LDFLAGS"] += f" -L{lib_dir} -lcpufeatures"
+        return env
+
+    def should_build(self, arch):
+        return Recipe.should_build(self, arch)
+
+    def build_arch(self, arch):
+        super().build_arch(arch)
+        build_dir = self.get_build_dir(arch.arch)
+        install_dir = join(build_dir, 'install')
+        ensure_dir(install_dir)
+        env = self.get_recipe_env(arch)
+
+        lib_dir = self.ctx.get_libs_dir(arch.arch)
+        png_include = self.get_recipe('png', self.ctx).get_build_dir(arch.arch)
+        freetype_inc = join(self.get_recipe('freetype', self.ctx).get_build_dir(arch), "include")
+
+        with current_directory(build_dir):
+
+            cpufeatures_dir = join(self.ctx.ndk.ndk_dir, "sources/android/cpufeatures")
+            lib_file = join(cpufeatures_dir, "obj", "local", arch.arch, "libcpufeatures.a")
+
+            if not exists(lib_file):
+                shprint(
+                    sh.Command(join(self.ctx.ndk_dir, "ndk-build")),
+                    f"NDK_PROJECT_PATH={cpufeatures_dir}",
+                    f"APP_BUILD_SCRIPT={cpufeatures_dir}/Android.mk",
+                    f"APP_ABI={arch.arch}",
+                    "APP_PLATFORM=latest",
+                    _env=env
+                )
+
+            shprint(sh.meson, 'setup', 'builddir',
+                    '--cross-file', join("/tmp", "android.meson.cross"),
+                    f'--prefix={install_dir}',
+                    '-Dpng=enabled',
+                    '-Dzlib=enabled',
+                    '-Dglib=disabled',
+                    '-Dgtk_doc=false',
+                    '-Dsymbol-lookup=disabled',
+
+                    # deps
+                    f'-Dpng_include_dir={png_include}',
+                    f'-Dpng_lib_dir={lib_dir}',
+                    f'-Dfreetype_include_dir={freetype_inc}',
+                    f'-Dfreetype_lib_dir={lib_dir}',
+                    _env=env)
+
+            shprint(sh.ninja, '-C', 'builddir', '-j', str(cpu_count()), _env=env)
+            # macOS fix: sometimes Ninja creates a dummy 'lib' file instead of a directory.
+            # So we remove and recreate the install directory using shell commands,
+            # since os.remove/os.makedirs behave inconsistently in this build env.
+            shprint(sh.rm, '-rf', install_dir)
+            shprint(sh.mkdir, install_dir)
+
+            shprint(sh.ninja, '-C', 'builddir', 'install', _env=env)
+
+
+recipe = LibCairoRecipe()

--- a/pythonforandroid/recipes/libcairo/meson.patch
+++ b/pythonforandroid/recipes/libcairo/meson.patch
@@ -1,0 +1,64 @@
+diff '--color=auto' -uNr cairo-1.18.4/meson.build cairo-1.18.4.mod/meson.build
+--- cairo-1.18.4/meson.build	2025-03-08 18:53:25.000000000 +0530
++++ cairo-1.18.4.mod/meson.build	2025-07-14 20:42:56.226164648 +0530
+@@ -235,11 +235,13 @@
+   conf.set('HAVE_ZLIB', 1)
+ endif
+ 
+-png_dep = dependency('libpng',
+-  required: get_option('png'),
+-  version: libpng_required_version,
+-  fallback: ['libpng', 'libpng_dep']
++png_inc = include_directories(get_option('png_include_dir'))
++png_lib = cc.find_library('png16', dirs: [get_option('png_lib_dir')], required: true)
++png_dep = declare_dependency(
++  include_directories: png_inc,
++  dependencies: [png_lib]
+ )
++
+ if png_dep.found()
+   feature_conf.set('CAIRO_HAS_SVG_SURFACE', 1)
+   feature_conf.set('CAIRO_HAS_PNG_FUNCTIONS', 1)
+@@ -265,7 +267,7 @@
+ 
+ # Disable fontconfig by default on platforms where it is optional
+ fontconfig_option = get_option('fontconfig')
+-fontconfig_required = host_machine.system() not in ['windows', 'darwin']
++fontconfig_required = false 
+ fontconfig_option = fontconfig_option.disable_auto_if(not fontconfig_required)
+ 
+ fontconfig_dep = dependency('fontconfig',
+@@ -304,11 +306,14 @@
+ freetype_required = host_machine.system() not in ['windows', 'darwin']
+ freetype_option = freetype_option.disable_auto_if(not freetype_required)
+ 
+-freetype_dep = dependency('freetype2',
+-  required: freetype_option,
+-  version: freetype_required_version,
+-  fallback: ['freetype2', 'freetype_dep'],
++freetype_inc = include_directories(get_option('freetype_include_dir'))
++freetype_lib = cc.find_library('freetype', dirs: [get_option('freetype_lib_dir')], required: true)
++
++freetype_dep = declare_dependency(
++  include_directories: freetype_inc,
++  dependencies: [freetype_lib]
+ )
++
+ if freetype_dep.found()
+   feature_conf.set('CAIRO_HAS_FT_FONT', 1)
+   built_features += [{
+diff '--color=auto' -uNr cairo-1.18.4/meson.options cairo-1.18.4.mod/meson.options
+--- cairo-1.18.4/meson.options	2025-03-08 18:53:25.000000000 +0530
++++ cairo-1.18.4.mod/meson.options	2025-07-14 20:43:00.473191452 +0530
+@@ -28,3 +28,11 @@
+ # Documentation
+ option('gtk_doc', type : 'boolean', value : false,
+        description: 'Build the Cairo API reference (depends on gtk-doc)')
++
++# Deps
++
++option('png_include_dir', type: 'string', value: '', description: 'Path to PNG headers')
++option('png_lib_dir', type: 'string', value: '', description: 'Path to PNG library')
++option('freetype_include_dir', type: 'string', value: '', description: 'Path to FreeType headers')
++option('freetype_lib_dir', type: 'string', value: '', description: 'Path to FreeType library')
++

--- a/pythonforandroid/recipes/pycairo/__init__.py
+++ b/pythonforandroid/recipes/pycairo/__init__.py
@@ -1,0 +1,26 @@
+from pythonforandroid.recipe import MesonRecipe
+from os.path import join
+
+
+class PyCairoRecipe(MesonRecipe):
+    version = '1.28.0'
+    url = 'https://github.com/pygobject/pycairo/releases/download/v{version}/pycairo-{version}.tar.gz'
+    name = 'pycairo'
+    site_packages_name = 'cairo'
+    depends = ['libcairo']
+    patches = ["meson.patch"]
+
+    def build_arch(self, arch):
+
+        include_path = join(self.get_recipe('libcairo', self.ctx).get_build_dir(arch), "install", "include", "cairo")
+        lib_path = self.ctx.get_libs_dir(arch.arch)
+
+        self.extra_build_args += [
+            f'-Csetup-args=-Dcairo_include={include_path}',
+            f'-Csetup-args=-Dcairo_lib={lib_path}',
+        ]
+
+        super().build_arch(arch)
+
+
+recipe = PyCairoRecipe()

--- a/pythonforandroid/recipes/pycairo/meson.patch
+++ b/pythonforandroid/recipes/pycairo/meson.patch
@@ -1,0 +1,24 @@
+diff '--color=auto' -uNr pycairo-1.28.0/cairo/meson.build pycairo-1.28.0.mod/cairo/meson.build
+--- pycairo-1.28.0/cairo/meson.build	2025-04-15 00:22:30.000000000 +0530
++++ pycairo-1.28.0.mod/cairo/meson.build	2025-07-14 21:56:34.782983845 +0530
+@@ -28,7 +28,10 @@
+   fs.copyfile(python_file, python_file)
+ endforeach
+ 
+-cairo_dep = dependency('cairo', version: cair_version_req, required: cc.get_id() != 'msvc')
++cairo_dep = declare_dependency(
++  include_directories: include_directories(get_option('cairo_include')),
++  link_args: ['-L' + get_option('cairo_lib'), '-lcairo']
++)
+ 
+ if cc.get_id() == 'msvc' and not cairo_dep.found()
+   if cc.has_header('cairo.h')
+diff '--color=auto' -uNr pycairo-1.28.0/meson_options.txt pycairo-1.28.0.mod/meson_options.txt
+--- pycairo-1.28.0/meson_options.txt	2025-04-15 00:22:30.000000000 +0530
++++ pycairo-1.28.0.mod/meson_options.txt	2025-07-14 21:56:52.824191314 +0530
+@@ -1,3 +1,5 @@
+ option('python', type : 'string', value : 'python3')
+ option('tests', type : 'boolean', value : true, description : 'build unit tests')
+ option('wheel', type : 'boolean', value : false, description : 'build for a Python wheel')
++option('cairo_include', type: 'string', value: '', description: 'Path to cairo headers')
++option('cairo_lib', type: 'string', value: '', description: 'Path to cairo libraries')

--- a/setup.py
+++ b/setup.py
@@ -21,7 +21,7 @@ data_files = []
 # https://github.com/kivy/buildozer/issues/722
 install_reqs = [
     'appdirs', 'colorama>=0.3.3', 'jinja2',
-    'sh>=2, <3.0; sys_platform!="win32"',
+    'sh>=2, <3.0; sys_platform!="win32"', 'meson', 'ninja',
     'build', 'toml', 'packaging', 'setuptools', 'wheel~=0.43.0'
 ]
 # (build and toml are used by pythonpackage.py)

